recipes/libcurl/all/conanfile.py

```diff
@@ -68,8 +70,10 @@ def imports(self):
             self.copy("*.dylib*", dst=self._source_subfolder, keep_path=False)
 
     def config_options(self):
-        if self.settings.os != "Macos":
+        if self.settings.os not in ["Macos", "iOS"]:
```

Comment:
Can this and other occurrences be replaced by the following code?
```suggestion
        if not tools.is_apple_os(self.settings.os):
```

Reply:
referring to the documents, this should work, good catch! 
will update the PR